### ISSUE TITLE
Back button on subsequent pages

### ISSUE THREAD

**Issue by jeffshaver** [mtr-cherish/cherish#124]

**Comment by weslers** [comment]
what pages should this button appear on?

**Comment by jeffshaver** [comment]
I ported this from trello. But I really don't understand the point of this... Just use the browser back button? Every browser has a back button....

**Comment by ZachStoltz** [comment]
I agree that back buttons would prob be a bad idea to be added for page navigation.

**Comment by iamlasse** [comment]
It was for when on mobile
